crankers/src/bin/main.rs

```diff
@@ -0,0 +1,138 @@
+use std::{collections::HashMap, path::PathBuf, time::Duration};
+
+use anyhow::{anyhow, Context};
+use clap::{arg, Parser};
+use jito_bytemuck::AccountDeserialize;
+use jito_vault_core::{vault::Vault, vault_operator_delegation::VaultOperatorDelegation};
+use jito_vault_cranker::{restaking_handler::RestakingHandler, vault_handler::VaultHandler};
+use log::info;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::{pubkey::Pubkey, signature::read_keypair_file};
+
+#[derive(Parser)]
+struct Args {
+    /// RPC URL for the cluster
+    #[arg(short, long, env, default_value = "https://api.devnet.solana.com")]
+    rpc_url: String,
+
+    /// Path to keypair used to pay
+    #[arg(short, long, env = "KEYPAIR_PATH")]
+    keypair: PathBuf,
+
+    /// Vault program ID (Pubkey as base58 string)
+    #[arg(
+        long,
+        env,
+        default_value = "Vau1t6sLNxnzB7ZDsef8TLbPLfyZMYXH8WTNqUdm9g8"
+    )]
+    vault_program_id: Pubkey,
```

Comment:
does this parse strings from cli properly?

Reply:
yep this works, same pattern used in stakenet for example:
```rust
    /// Steward program ID
    #[arg(
        long,
        env,
        default_value = "Stewardf95sJbmtcZsyagb2dg4Mo8eVQho8gpECvLx8"
    )]
    pub steward_program_id: Pubkey,
```